rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy

```diff
@@ -3230,6 +3230,8 @@ class ExecutionService implements ApplicationContextAware, StepExecutor, NodeSte
         }
         result.sourceResult = wresult
 
+        Map<String, Map<String, String>> data = ((WorkflowExecutionResult)wresult).getSharedContext().getData(ContextView.global())
+        executionContext.getOutputContext().addOutput(ContextView.global(),data)
```

Comment:
I think we should restrict this, to perhaps only `export.*`, so that the user can explicitly export variables (by renaming via the Uplift plugin)